src/core/containers.py

```diff
@@ -34,3 +34,25 @@ def __init__(self, flow_name, task_name):
 
     def as_dict(self):
         return {"flow": self.flow, "task": self.task}
+
+
+class Populate(Container):
+    __slots__ = ["path", "validators", "value"]
+
+    def __init__(self, path="", validators="", value="", **kwargs):
+        super().__init__(**kwargs)
+        self.path = path
+        self.validators = validators
```

Comment:
Going to need a mechanism like the components to extract the validators and we will have to add to the input component
```Input.get_validators``` func to extract it.